raft/src/test/java/org/apache/kafka/snapshot/SnapshotWriterReaderTest.java

```diff
@@ -151,6 +194,57 @@ public void testAppendToFrozenSnapshot() throws Exception {
         );
     }
 
+    private int validateDelimiters(
+        RawSnapshotReader snapshot,
+        long lastContainedLogTime
+    ) {
+        assertNotEquals(0, snapshot.sizeInBytes());
+
+        int countRecords = 0;
+
+        Iterator<RecordBatch> recordBatches = Utils.covariantCast(snapshot.records().batchIterator());
+
+        assertEquals(Boolean.TRUE, recordBatches.hasNext());
```

Comment:
You can use `assertTrue` instead. This comment applies to a few places.

Reply:
thanks